frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/utils.ts

```diff
@@ -1,9 +1,27 @@
 import * as _ from 'lodash';
-import { TektonParam } from '../../../types';
+import { getResourceModelFromTaskKind } from '../../../utils/pipeline-augment';
+import { PipelineKind, PipelineTask, TektonParam } from '../../../types';
+import { ResourceModelLink } from '../resource-overview/DynamicResourceLinkList';
 
 export const removeEmptyDefaultFromPipelineParams = (parameters: TektonParam[]): TektonParam[] =>
   _.map(
     parameters,
     (parameter) =>
       _.omit(parameter, _.isEmpty(parameter.default) ? ['default'] : []) as TektonParam,
   );
+
+export const getPipelineTaskLinks = (pipeline: PipelineKind) => {
+  const tasks = [pipeline.spec.tasks, pipeline.spec.finally];
```

Comment:
Please type the return. Something like this maybe (for clarity on each sub array)?

```suggestion
type Tasks = PipelineTask[];
type FinallyTasks = PipelineTask[];
type PipelineTaskLinks = [Tasks, FinallyTasks];
export const getPipelineTaskLinks = (pipeline: PipelineKind): PipelineTaskLinks => {
  const tasks: PipelineTaskLinks = [pipeline.spec.tasks, pipeline.spec.finally];
```

Reply:
After talking with Aritra over this... I realized that I actually typed this waaay wrong lol. But hey, he figured it out, that's all that matters 🙂

Reply:
Updated it to use function instead of array manipulation